docs/customizations/helm.rst

```diff
@@ -289,7 +293,7 @@ SR-IOV Network Operator Helm chart customization options can be found `here <htt
      -
    * - sriov-network-operator.images.operator
      - string
-     - `"nvcr.io/nvidia/mellanox/sriov-network-operator:network-operator-24.10.0"`
+     - `"nvcr.io/nvstaging/mellanox/sriov-network-operator:network-operator-24.10.0-rc.1"`
```

Comment:
Why was this changes?

Reply:
This is what configured in master branch, the generation is done automatically.